src/modules/OpenChannelSettings/components/ParticipantUI/ParticipantItem.tsx

```diff
@@ -65,32 +65,31 @@ export const UserListItem: React.FC<UserListItemProps> = ({
             </>
           )}
           menuItems={(closeDropdown) => (
-            <MenuItems
-              openLeft
-              parentRef={avatarRef}
-              // for catching location(x, y) of MenuItems
-              parentContainRef={avatarRef}
-              // for toggling more options(menus & reactions)
-              closeDropdown={closeDropdown}
-              style={{ paddingTop: '0px', paddingBottom: '0px' }}
-            >
-              {
-                renderUserProfile
-                  ? renderUserProfile({
-                    user: user,
-                    currentUserId: currentUser,
-                    close: closeDropdown,
-                  })
-                  : (
-                    <UserProfile
-                      disableMessaging
-                      user={user}
-                      currentUserId={currentUser}
-                      onSuccess={closeDropdown}
-                    />
-                  )
-              }
-            </MenuItems>
+            renderUserProfile
+              ? renderUserProfile({
+                user: user,
+                currentUserId: currentUser,
+                close: closeDropdown,
+                avatarRef,
+              })
+              : (
+                <MenuItems
+                  openLeft
+                  parentRef={avatarRef}
+                  // for catching location(x, y) of MenuItems
+                  parentContainRef={avatarRef}
+                  // for toggling more options(menus & reactions)
+                  closeDropdown={closeDropdown}
+                  style={{ paddingTop: '0px', paddingBottom: '0px' }}
```

Comment:
I see repeating
```
{ paddingTop: '0px', paddingBottom: '0px' }
```
can we make this default? it not why not?

Reply:
That's a good suggestion, but I'm little bit worried about the side effects, let me do it in a separate PR